javascript/src/handleDSRWebhookPaginated.js

```diff
@@ -76,6 +77,7 @@ async function scheduleAccessChunkedRequest(
     }
     logger.info(`Successfully uploaded data - ${requestLink}`);
   } catch (error) {
+    logger.info('Handle DSR paginated error');
```

Comment:
Why is this an info log? Also, why not just have this code in the Typescript scripts too?

Reply:
Just adding to debug it for staging. I want to make sure that this is the route that is hit in the logs. Don't need to add more logs on the the one that we are running locally

Reply:
Will remove this once the fix is working